src/app-styles.ts

```diff
@@ -0,0 +1,5 @@
+import { styled } from 'styled-components';
+
+export const Container = styled.div`
+  margin: 0 400px;
```

Comment:
uma coisinha que evitamos nos projetos e seria legal já ir se acostumando, é o uso de `magicNumbers`(Link abaixo falando um pouco mais o que é):
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Basicamente os magic numbers são números usados diretamente no estilo e na lógica, ao invés de usar eles diretamente geralmente definimos uma const com um nome que expresse o que ele representa, como no seu caso seria: 

```suggestion
const MARGIN_SIDE = 400
export const Container = styled.div`
  margin: 0 ${MARGIN_SIDE}px;

```